docs/api/4.4.md

```diff
@@ -3200,6 +3200,40 @@ When defining validation rules for fields of type `DateTime`, the following plac
     }
     ```
 
+### `DateTime` validation
```

Comment:
this doesn't look like it belongs here - this heading also above at L3110

Reply:
Oops. That was a copy/paste gone wrong. Should be `Media`, not `DateTime`. Updated!